sql.txt

```diff
@@ -1,14 +1,26 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+    SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+    SELECT * FROM Customers ORDER BY ContactName ASC;
```

Comment:
So the answer here is incorrect because it returns all the Customers from the table, but orders them in ascending order. Try to find a way to get only the customers that that have a contact name that starts with a.

Reply:
I realized I read it too fast and assumed something else.
`SELECT * FROM Customers where ContactName < "B";

        `